core/domain/skill_services.py

```diff
@@ -848,30 +876,35 @@ def get_skill_mastery(user_id, skill_id):
             requested.
 
     Returns:
-        degree_of_mastery: float. Mastery degree of the user for the
-            requested skill.
+        degree_of_mastery: float or None. Mastery degree of the user for the
+            requested skill, or None if UserSkillMasteryModel does not exist
+            for the skill.
     """
     model_id = user_models.UserSkillMasteryModel.construct_model_id(
         user_id, skill_id)
-    degree_of_mastery = user_models.UserSkillMasteryModel.get(
-        model_id).degree_of_mastery
+    user_skill_mastery_model = user_models.UserSkillMasteryModel.get(
+        model_id, strict=False)
 
-    return degree_of_mastery
+    if not user_skill_mastery_model:
+        return None
+    return user_skill_mastery_model.degree_of_mastery
 
 
 def get_multi_skill_mastery(user_id, skill_ids):
```

Comment:
should we rename this to `get_multi_user_skill_mastery` similar to `create_multi_user_skill_master`? (or change the later name to have consistency.)

Reply:
Done.